packages/shared/lib/services/sync/orchestrator.service.ts

```diff
@@ -438,17 +499,19 @@ export class OrchestratorService {
             const { providerConfigKey } = flow;
             const name = flow.name || flow.syncName;
 
-            await this.create(
+            await this.createSyncForConnections(
                 existingConnections as Connection[],
                 name as string,
                 providerConfigKey,
                 environmentId,
                 flow as unknown as IncomingFlowConfig,
                 logContextGetter,
+                orchestrator,
                 false
             );
         }
     }
+    // TODO:
```

Comment:
I'll take care of returning the correct sync data depending on the flag in next PR